server.go

```diff
@@ -27,7 +27,7 @@ var (
 	registerToken = flag.String("register_token", "", "Register with the master and get certificates")
 	masterCert    = flag.String("master_cert", "%rufs_var_storage%/master/ca.crt", "Path to ca file of the master")
 
-	fileCacheMtx sync.Mutex
+	fileCacheMtx sync.RWMutex
```

Comment:
I think this is a performance degradation: RWMutex has more overhead than a regular Mutex, and I don't think there's actually be much contention resolved by allowing multiple readers.